src/test/java/com/wb/WeatherForecastApiApplicationUnitTests.java

```diff
@@ -0,0 +1,85 @@
+package com.wb;
+
+import static org.junit.Assert.assertEquals;
```

Comment:
Inna rzeczą jest to że próbujesz testowac po raz kolejny kontroler. Dla kontrolera testy już masz w testach integracyjnych. Sugerowałem żebys tutaj przetestował `WeatherService`. Podpowiedz:
1. Używasz junit 5
2. Spring Runner bo chcesz miec prawdziwy serwis
3. Mockujesz odpowiedz z serwera pogodowego żeby zwracał ci to co chcesz (tak na prawde to symulujesz odpowiedz z tego serwera, nigdy do niego nie strzelasz) 
4.  Testujesz czy serwis umie przetworzyc i zmapowac to co otrzymal z zamockowanego serwera
Jeśli zerkniesz w ten link to w punkcie 3.3 masz jak na dłoni jak to powinno wyglądac. `DetailsServiceClient` to u Cieibe `WeatherService`